ffi/derive/src/convert.rs

```diff
@@ -265,6 +277,136 @@ fn derive_try_from_repr_c_for_opaque_item_vec(input: &DeriveInput) -> TokenStrea
     }
 }
 
+fn derive_try_from_repr_c_for_transparent_item(
```

Comment:
just noticed this file doesn't have any doc comments at all

Reply:
It doesn't have to. Anything that stays in the `derive` crate is almost completely for internal use only.